### PR TITLE
feat(SectScans,LunarScans): create connectors

### LINKED ISSUE #6295: [Site Request] Lunar Scans
**Name of the website**
Lunar Scans

**Website urls (examples below)**
- Site: https://lunarscan.org/
- Manga List: https://lunarscan.org/series/
- Manga example: https://lunarscan.org/series/my-sisters-friends/
- Chapter online viewer example: https://lunarscan.org/my-sisters-friends-chapter-1/

**Languages**
English

**Website relationship*
I'm not sure

**Additional details**
MangaStream Wordpress Theme

### LINKED ISSUE #6296: [Site Request] SectScans.com
**Name of the website**
SectScans.com

**Website urls (examples below)**
- Site: https://sectscans.com
- Manga List: https://sectscans.com/comics/
- Manga example: https://sectscans.com/comics/the-son-of-the-first-emperor-kills-enemies-and-becomes-a-god/
- Chapter online viewer example: https://sectscans.com/comics/the-son-of-the-first-emperor-kills-enemies-and-becomes-a-god/chapter-49/

**Languages**
English

**Website relationship*
No Website relationship.

**Additional details**
It uses Madara Theme

### PR DESCRIPTION
**Pull request by paulphyo**
Fixes https://github.com/manga-download/hakuneko/issues/6295
Fixes https://github.com/manga-download/hakuneko/issues/6296

Side-note: Lunarscan does take a substantial amount of time to download but it could just be my internet xd
Do let me know if there's anything I can fix ^

### CODE REVIEW

File: src/web/mjs/connectors/LunarScans.mjs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import WordPressMangastream from "./templates/WordPressMangastream.mjs";
+
+export default class LunarScans extends WordPressMangastream {
+  constructor() {
+    super();
+    super.id = "lunarscans";
+    super.label = "Lunar Scans";
+    this.tags = ["webtoon", "english", "scanlation", "hentai"];
+    this.url = "https://lunarscan.org/";
```
Comment (MikeZeDev) [inline src/web/mjs/connectors/LunarScans.mjs]: Remove trailing / on url and put it before the path. Its like that on other connectors ;)

    this.url = "https://lunarscan.org";
    this.path = "/series/list-mode/";

File: src/web/mjs/connectors/SectScans.mjs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import WordPressMadara from "./templates/WordPressMadara.mjs";
+
+export default class SectScans extends WordPressMadara {
+  constructor() {
+    super();
+    super.id = "sectscans";
+    super.label = "Sect Scans";
+    this.tags = ["manga", "english", "webtoon", "scanlation"];
+    this.url = "https://sectscans.com/";
```
Comment (MikeZeDev) [inline src/web/mjs/connectors/SectScans.mjs]: Remove trailing backlash on url.

**Review by MikeZeDev — APPROVED** [review]
Minor changes requested. It works, Thanks .

**Comment by MikeZeDev** [comment]
https://github.com/manga-download/hakuneko/actions/runs/6446809979/job/17503184007?pr=6301 

May want to fix Lint errors.  ;)

**Comment by paulphyo** [comment]
> [manga-download/hakuneko/actions/runs/6446809979/job/17503184007?pr=6301](https://github.com/manga-download/hakuneko/actions/runs/6446809979/job/17503184007?pr=6301)
> 
> May want to fix Lint errors. ;)

oops, mb

**Comment by paulphyo** [comment]
Mb i forgot that step xd It should be fixed now

**Pull request merged**